tests/snappi/conftest.py

```diff
@@ -76,3 +76,30 @@ def enable_packet_aging_after_test(duthosts, rand_one_dut_hostname):
 
     duthost = duthosts[rand_one_dut_hostname]
     enable_packet_aging(duthost)
+
+
+@pytest.fixture(scope="module")
+def retrieve_pfc_pause_quanta(duthosts, rand_one_dut_hostname):
+    """
+    Retrieves a dictionary of pfc pause quantas for the headroom test
+    Args:
+        duthost (Ansible host instance): device under test
+    Returns:
+        pfc_pause_quanta_values: Mapped from pfc pause quanta to whether
+                                the headroom test will fail or not
+                                E.g. {1:True, 2:False, 3:False}
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    platform = duthost.facts['platform']
+    pfc_pause_quanta_values = {}
+    if '8102' in platform:
+        pfc_pause_quanta_values[1] = True
```

Comment:
How are you setting the delay values? I only see True and False

Reply:
The delay values are integers, and is being populated into this dictionary from which the pfc delay is being populated for the config in line 75 in the primary test case file.

Reply:
The key, value pair represents whether the headroom value will not be exceeded or not for that delay quanta; True means it will not be exceeded

Reply:
Probably I am missing something. Where is the integer value specified?

Reply:
This is a dictionary with a key value pair in the form int, bool; so for instance, I have pfc_pause_quanta_values[1] = True; meaning for a delay quanta of 1, the headroom limit will not be exceeded. Another one is pfc_pause_quanta_values[3] = False; meaning the headroom limit will be exceeded for a delay quanta of 3.

Reply:
Can you add some comments in there to indicate that the array index is actually your delay quanta?

Reply:
It seems to me that pfc_pause_quanta_values should be integers between 0 and 65535 and 0 represents "resume".